app/lib/constants.js

```diff
@@ -9,6 +9,16 @@ module.exports = {
       open: 'open',
     },
   },
+  businessHours: {
+    end: {
+      hour: 18,
+      minute: 0,
+    },
+    start: {
+      hour: 8,
+      minute: 0,
+    },
+  },
```

Comment:
Should the business hours/minutes be exposed as env vars so they can be easily changed without a code change?

Reply:
I did consider this but didn't do. I've added it in now.